### ISSUE TITLE
Add a blue button for "Manage release"

### ISSUE THREAD

**Issue by mejackreed** [sul-dlss/argo#805]
Requires SDR admin role or APO manager role
If a collection:
```
Manage release to discovery applications for collection sj775xm6965
       Douglas Menuez Photography, circa 1986-2003
Currently released to Searchworks[, Target2 ...]
-------
Release to : [Searchworks] <-- pulldown list for available targets
( * ) Release this collection and all it's members *
(  ) Release only this collection description but not any of its members (members previously released will be withdrawn) <-- needed?
(  ) Withdraw this collection from release * <-- only offered if there is a current "true" for the selected target
* Any members that have their own overriding release information will not be affected
```
If an item:
```
Manage release to discovery applications for item bb423jr2180
       The complaint, and the consolation: or, Night thoughts
Currently released to Searchworks[,target 2...] as part of collection "Douglas Menuez Photography, circa 1986-2003"
Release to : [Searchworks] <-- pulldown list for available targets
( * ) Release this item
(  ) Withdraw this item from release  <-- only offered if there is a current "true" for the selected target either locally or in its collection
```
Consider the release of an item that is already released by a collection a no-op?  Or allow if to create a local release tag in case it's an effort to white-list items before withdrawing the collection? To start, assume the creation of a local tag regardless of collection status.
Confirmation information, an opportunity to say things might take time for large collections as things work through workflow.

```
New release settings saved for Searchworks 
Please leave time for affected items to be republished in PURL and processed by the Searchworks indexer. 
```

**Comment by mejackreed** [comment]
This requirement:

```
(  ) Withdraw this item from release  <-- only offered if there is a current "true" for the selected target either locally or in its collection
```
Feels problematic to me. Can't we just do what we are doing elsewhere and offer:

```
Do not release this item*
```

**Comment by LynnMcRae** [comment]
It's a but complicated with something you are withdrawing vs something you are white or blacklisting ahead of time, but I think the answer here is the same as for the bulk actions dialog ... uniform simple options, with perhaps a bit wordy footnote.  Let me give it a try.
